play/dbt-materialize/dbt/adapters/materialize/connections.py

```diff
@@ -0,0 +1,62 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from dbt.adapters.postgres import PostgresConnectionManager
+from dbt.adapters.postgres import PostgresCredentials
+
+import dbt.exceptions
+from dataclasses import dataclass
+from dbt import flags
+from dbt.logger import GLOBAL_LOGGER as logger
+
+
+@dataclass
+class MaterializeCredentials(PostgresCredentials):
+    @property
+    def type(self):
+        return "materialize"
+
+
+class MaterializeConnectionManager(PostgresConnectionManager):
+    TYPE = "materialize"
+
+    @classmethod
+    def open(cls, connection):
+        connection = super().open(connection)
+        # Prevents psycopg connection from automatically opening transactions
+        # More info: https://www.psycopg.org/docs/usage.html#transactions-control
+        connection.handle.autocommit = True
```

Comment:
If this is to workaround the `SHOW VIEWS` issue, I believe that was fixed. However, if there are other commands that do not work inside of transactions, let's keep it!

Reply:
This is actually a workaround for all of the other commands executed against Materialize! (ex: `CREATE`, `DROP`)